src/status_im2/common/scan_qr_code/view.cljs

```diff
@@ -35,13 +35,14 @@
    [quo/text
     {:size   :heading-1
      :weight :semi-bold
-     :style  style/header-text}
+     :style  (style/header-text bottom-padding?)}
```

Comment:
Introducing a new prop `bottom-padding?` is a bit overkill. It's affecting other places too.

![PR](https://github.com/status-im/status-mobile/assets/19339952/51090de6-62f6-4ade-a6dd-cec435cb386f)

You can use the existing `subtitle` prop to check if the subtitle exists and add the padding.